nixos/modules/services/amqp/rabbitmq.nix

```diff
@@ -178,9 +204,9 @@ in {
         RABBITMQ_CONFIG_FILE = config_file;
         RABBITMQ_PLUGINS_DIR = concatStringsSep ":" cfg.pluginDirs;
         RABBITMQ_ENABLED_PLUGINS_FILE = pkgs.writeText "enabled_plugins" ''
-          [ ${concatStringsSep "," cfg.plugins} ].
+          [ ${concatStringsSep "," (cfg.plugins ++ optional cfg.managementPlugin.enable "rabbitmq_management")} ].
```

Comment:
Could you also do something like `systemd.services.rabbitmq.plugins = optional cfg.managementPlugin.enable [ "rabbitmq_management" ]`? Then the list type of the option takes care of merging. Not sure what's better though.

Reply:
Thank you for the review!

I'm not sure what you mean here.
There is no systemd.services.rabbitmq.plugins
The plugins is an option coming from the configuration. I thought about adding it to default, however, somebody adding plugins would override the default. I'm open to changing this, I just don't see how.

Reply:
`systemd.` part is probably a typo. You can change any piece configuration, like it happens with `services.rabbitmq.configItems` assignment nearby. If you add  the proposed `services.rabbitmq.plugins` assignment there, explicit list concatenation with `++` will be unnecessary.

Reply:
Ah sorry, I meant [`services.rabbitmq.plugins`](https://search.nixos.org/options?channel=21.05&show=services.rabbitmq.plugins&from=0&size=50&sort=relevance&type=packages&query=rabbitmq).

Reply:
Oh I see, adding it under `config`, I thought you were talking about `options`.
Nice trick. I'm pushing the change and testing it.
Thank you!